client/src/modules/dashboard/Chapters/pages/ChaptersPage.tsx

```diff
@@ -21,9 +21,6 @@ export const ChaptersPage: NextPage = () => {
   if (isLoading || error)
     return <DashboardLoading loading={isLoading} error={error} />;
 
-  // TODO: render something nicer if this happens. A 404 page?
-  if (!data.chapters) return <div> No Chapters found</div>;
-
```

Comment:
This is not needed. When there's no chapters in the instance, `data.chapter` exists, but it's an empty array. In such case, the `DataTable` will display _Empty_ text. For `data.chapter` not exist I assume there would need to be some kind of error, which would be caught either as `error` or event earlier.